spec/models/author_spec.rb

```diff
@@ -75,22 +76,19 @@
 
   describe ".bibliography" do
     it "Returns a human readable string containing an Author's bibliography" do
+      author = create(:author, name: "James Baldwin", biography: "James Baldwin was an American novelist, playwright, essayist, poet, and activist.", publisher_name: "Penguin Books", publisher_email: "info@penguin.com")
+      drama = create(:genre, name: "Drama")
+      comedy = create(:genre, name: "Comedy")
 
-      author = Author.create!(name: "James Baldwin", biography: "James Baldwin was an American novelist, playwright, essayist, poet, and activist.", publisher_name: "Penguin Books", publisher_email: "info@penguin.com")
-      expect(author.bibliography).to eq("")
-
-      drama = Genre.create!(name: "Drama")
-      comedy = Genre.create!(name: "Comedy")
-
-      Book.create!(name: "Emma", isbn: 9292929292, created_at: Date.new(2000, 1, 1), authors: [author], genres: [drama, comedy])
+      create(:book, name: "Emma", isbn: 9292929292, created_at: Date.new(2000, 1, 1), authors: [author], genres: [drama, comedy])
```

Comment:
do we need the ISBN here?

Reply:
Nope! Have removed all this extra stuff.